site/cypress/e2e/gradeables/gradeable_results.spec.js

```diff
@@ -0,0 +1,51 @@
+
+
+describe('Testing functionality of Autograder Results', () => {
+    beforeEach(() => {
+        cy.visit(['sample','gradeable','closed_team_homework']);
+        cy.login('student');
+
+    });
+
+    it('Should display confetti', () => {
+        cy.get('#confetti_canvas[style="display: block;"]').should('not.exist');
+        cy.get('div[onclick="addConfetti();"]').click();
+        cy.get('#confetti_canvas[style="display: block;"]').should('exist');
+    });
+
+    it('Should show and hide details', () => {
+        //Open
+        cy.get('#testcase_1').should('have.css', 'display').and('match', /none/);
+        cy.get('.loading-tools-show').eq(1).click();
+        cy.get('#testcase_1').should('have.css', 'display').and('not.match', /none/);
+        //Close
+        cy.get('.loading-tools-hide').first().click();
+        cy.get('#testcase_1').should('have.css', 'display').and('match', /none/);
```

Comment:
We should not be targeting things based on CSS properties for any reason.  It would be better to use `.should('exist')` or .should('be.visible')` here.  Apply throughout.

Reply:
done